core/src/test/java/org/jdbi/v3/core/statement/TestQueries.java

```diff
@@ -275,8 +274,8 @@ public void testFirstWithNoResult() {
     @Test
     public void testNullValueInColumn() {
         h.execute("insert into something (id, name) values (?, ?)", 1, null);
-        Optional<String> s = h.createQuery("select name from something where id=1").mapTo(String.class).findFirst();
-        assertThat(s.isPresent()).isFalse();
+        String s = h.createQuery("select name from something where id=1").mapTo(String.class).first();
+        assertThat(s).isNull();
```

Comment:
This changes test semantics. 

BTW What about testing all cases:
```
h.createQuery("....").mapTo(String.class).first(); // .. should be null
h.createQuery("....").mapTo(String.class).one(); // .. should be null
h.createQuery("....").mapTo(String.class).findOne(); // .. should be Optional.empty
```

?

Reply:
It does change semantics--by switching from `findFirst()` to `first()` we're getting more specific, and requiring the result to contain a single row that is null. The old test could conflate no rows with a single null row.